htdocs/takepos/takepos.php

```diff
@@ -58,7 +58,7 @@
 $categorie = new Categorie($db);
 $categories = $categorie->get_full_arbo('product');
 ?>
-var categories = JSON.parse( '<?php echo json_encode($categories);?>' );
+var categories = <?php echo json_encode($categories, JSON_HEX_APOS); ?>;
```

Comment:
Note that there is an absctraction layer when json_encode is not available on php into json.lib.php with a

if (! function_exists('json_encode')) ...

And in this case, the second parameter is not supported, so your new code may fails.

Reply:
@eldy 
we have errors when the category label contains a simple quote

Reply:
@eldy ok i remove option